base/shared/src/main/scala/scalaz/control/Inf.scala

```diff
@@ -0,0 +1,43 @@
+package scalaz
+package control
+
+/**
+  * Modelled from http://www.cse.chalmers.se/~nad/publications/danielsson-altenkirch-mixing.pdf
+  */
+sealed abstract class Bool
+sealed abstract class True extends Bool
```

Comment:
This Scaladoc comment will apply to `Bool`, which you probably did not intend.

Reply:
I've fixed this with more documentation in updates for tomorrow, thanks :)